torchx/schedulers/slurm_scheduler.py

```diff
@@ -557,7 +557,17 @@ def log_iter(
         )
 
     def list(self) -> List[str]:
-        raise NotImplementedError()
+        # By default sacct only returns accounting informations of jobs launched on the current day
+        # To return all jobs launched, set starttime to one second past unix epoch time
+        # Starttime will be modified when listing jobs by timeframe is supported
+        p = subprocess.run(
+            ["sacct", "--json", "-S1970-01-01-00:00:01"],
```

Comment:
Can this cause a problem if there are large number of jobs? Is there a notion of pagination?